content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,39 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+Let's say we want to print the filename also in the description and so we update
+the `process.py` as:
+
+```python
+print('Number of lines in %s:'%(sys.argv[1]))
```

Comment:
Still too complicated: Need to `import sys`, uses outdated `%` string formatting (f-string are the py3 standard), and it's still not well formatted (lacking proper spacing).

Reply:
> Need to import sys 

`sys` is already imported in the file.

>  uses outdated % string formatting 

I think most of the people still uses it as I have seen this format on various Kaggle competitions and open source projects. (although I would change it to f-string).

>  it's still not well formatted (lacking proper spacing).

I have checked this code with py-lint and its showing no errors with it. Please can you suggest a linter to check for these errors. Thanks

Reply:
I see `sys.argv[1]` is already used indeed. And you changed to f-string, so resolving.